frontend/src/app/commonComponents/CovidResultGuidance.tsx

```diff
@@ -143,16 +143,19 @@ const CovidResultGuidance = (props: Props) => {
       </p>
     );
   }
-  if (result === "UNDETERMINED" || result === "UNKNOWN") {
-    covidGuidanceArray.push(
-      setCovidResultInfo("UNDETERMINED", isPatientApp, t)
-    );
-  }
-  if (result !== "POSITIVE") {
-    covidGuidanceArray.push(setCovidResultInfo("NEGATIVE", isPatientApp, t));
-  }
-  if (result === "POSITIVE") {
-    covidGuidanceArray.push(setCovidResultInfo("POSITIVE", isPatientApp, t));
+
+  switch (result) {
+    case "UNDETERMINED":
```

Comment:
This seems to imply that we will no longer encounter `"UNKNOWN"` as a result here, is that the case?

Reply:
Yup, if "UNKNOWN" is encountered, do nothing